proj/vs2019/ophd.vcxproj

```diff
@@ -89,7 +93,7 @@
       <PrecompiledHeader />
       <WarningLevel>Level4</WarningLevel>
       <DebugInformationFormat>ProgramDatabase</DebugInformationFormat>
-      <EnablePREfast>false</EnablePREfast>
+      <EnablePREfast>true</EnablePREfast>

```

Comment:
For reference:
[VCCLCompilerTool.EnablePREfast Property](https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.vcprojectengine.vcclcompilertool.enableprefast?view=visualstudiosdk-2017)

> true to enable code analysis (/analyze).